books/refgram/src/concepts/time.md

```diff
@@ -25,29 +29,32 @@ update-context set-initial-span present-span
 
 (For the following explanations we'll consider it is the present)
 
-## How a simple sentence work?
+## How does a simple sentence work?
 ```
 mian buri meon
 cat  eat  apple
 ```
 
-This sentence is only true if there is an overlap between those 3 events:
+This sentence is only true if there is an overlap between the following three
+events:
+
 - a cat X exists
 - an apple Y exists
 - X eats Y
 
-This overlapping time span must itself overlap with the initial time span (present)
+This overlapping time span must itself overlap with the initial time span
+(present).
 
 ## Possible / Necessary?
-Tenses used in sentence each have 2 variants: possible and necessary.
+Tenses have two variants: possible and necessary.
 
-Possible means that there exist at least 1 time line in which the event occurs.
+Possible means that there exists at least one timeline in which the event occurs.
 
-Necessary means that it must occurs in all timelines.
+Necessary means that it must occur in all timelines.
 
 ## How to use tenses in a sentence?
 Tense predicates express that some proposition occurs in relation with the
-outer sentence __overlapping span__ (the time span that is part of the time
+outer sentence AKA __overlapping span__ (the time span that is part of the time
```

Comment:
"AKA"? I know "Also Known As", but it would not match what the sentence mean.
it is the overlapping span of the outer sentence.